instrumentation-genai/opentelemetry-instrumentation-openai-agents/examples/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenAI Agents Instrumentation Examples
```

Comment:
Currently all other genai instrumentations have a `manual` and `zerocode` examples. Perhaps we can follow up with a separate pr for zerocode. https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation-genai/opentelemetry-instrumentation-vertexai/examples